api/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6
```

Comment:
Let's note this version in the
- Github Wiki under a file called setup 
-  our developement document

Reply:
done